src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -25,4 +25,27 @@ public void equals_False_X10() {
     Integer i = Integer.valueOf(10);
     assertNotEquals(rnX, i);
   }
+
+  @Test
+  public void equals_True_Symmetric_X_VV() {
+    RomanNumeral rnVV = new RomanNumeral("VV");
+    assertEquals(rnX, rnVV);
+    assertEquals(rnVV, rnX);
+  }
+
+  @Test
+  public void equals_True_Transitive_X_VV_VIIIII() {
+    RomanNumeral rnVV = new RomanNumeral("VV");
+    RomanNumeral rnV5I = new RomanNumeral("VIIIII");
+    assertEquals(rnX, rnVV);
+    assertEquals(rnVV, rnV5I);
+    assertEquals(rnX, rnV5I);
+  }
+
+  //  @Test
```

Comment:
Please remove this.

Reply:
Do you meant the method I commented out?